content/en/docs/tutorials/services/connect-applications-service.md

```diff
@@ -302,6 +306,47 @@ kubectl get configmaps
 NAME             DATA   AGE
 nginxconfigmap   1      114s
 ```
+
+Details of the configmap ```nginxconfigmap``` can be viewed with the following command
+```shell
+kubectl describe cm nginxconfigmap
+
```

Comment:
There's a line missing here. Also, write `configmap` not `cm` - it helps people learn.

Reply:
Changed cm to conffigmap. Were you referring to a missing line after the kubectl describe cm nginxconfigmap
 line? That has been added as well

Reply:
I'd expect to see <tt>```</tt> on this line, ending the sample command.